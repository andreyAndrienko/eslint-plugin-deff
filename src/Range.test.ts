import { Range } from "./Range";

describe("range", () => {
  it("should instantiate with correct parameters", () => {
    const range: Range = new Range(0, 1);
    expect(range).toBeInstanceOf(Range);
  });

  it("should throw TypeError when parameters are flipped", () => {
    expect(() => new Range(1, 0)).toThrowErrorMatchingSnapshot();
  });
});
