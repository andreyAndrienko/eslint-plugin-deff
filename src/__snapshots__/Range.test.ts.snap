// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`range should throw TypeError when parameters are flipped 1`] = `"inclusiveLowerBound must be strictly less than exclusiveUpperBound: 1 >= 0"`;
